google-cloud-translate/lib/google-cloud-translate.rb

```diff
@@ -131,14 +116,10 @@ def translate key = nil, scope: nil, retries: nil, timeout: nil
     #   translation = translate.translate "Hello world!", to: "la"
     #   translation.text #=> "Salve mundi!"
     #
-    def self.translate key = nil, project_id: nil, credentials: nil, scope: nil,
-                       retries: nil, timeout: nil, project: nil, keyfile: nil
-      require "google/cloud/translate"
-      Google::Cloud::Translate.new key: key, project_id: project_id,
-                                   credentials: credentials,
-                                   scope: scope, retries: retries,
-                                   timeout: timeout,
-                                   project: project, keyfile: keyfile
+    def self.translate key = nil, project_id: nil, credentials: nil, scope: nil, retries: nil, timeout: nil
+      require "google/cloud/translate/v2"
```

Comment:
These should probably also default to V3 (and have a `:version` argument to select a version).

Reply:
I'm waiting to get all the samples working before taking another pass at the inline docs and the guides. My guess is we will show the V3 code examples before the V2 code examples.

Reply:
This has been updated with a v3 client example, but it isn't using `version: :v3`. PTAL.

Reply:
The v3 client doesn't support API Keys, only the legacy v2 client supports them.